docs/parameter tips

```diff
@@ -0,0 +1,42 @@
+# Tips for using 1.4 API Required Parameters
```

Comment:
The filename should be renamed to parameter_tips.md so it can be rendered properly as a markdown file.